ecommerce-ui/src/components/BnbShoppingCart.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useContext } from "react";
+import DisplayCard from "../UI/DisplayCard";
+import RentalsInCart from "./RentalsInCart";
+import RentalCartContext from "../store/RentalCartContext";
+import { Button, List, Typography } from "@mui/material";
+import PropTypes from "prop-types";
+
+function BnbShoppingCart({ displayCart }) {
+  const rentalCart = useContext(RentalCartContext);
+
+  const cartItems = rentalCart.rentals.map((rental, index) => {
+    return (
+      <RentalsInCart
+        key={index}
```

Comment:
Nice! Just as a pro-tip down the line you'll want to make the key something better than an index. Say each rental had a unique ID or something, that'd be better or even doing something like this is just a bit better.  Just to keep things as unique as possible.

`<li key={`rental-${id}`} />`

Reply:
Ahhh great to know, thanks!